docs/core/reports/customisation.md

```diff
@@ -83,6 +83,22 @@ Then this will be displayed at the top of reports:
 
 Note that you can also specify a path to a config file using `-c`.
 
+## Listing software versions
+
+For some modules the software version can be parsed from the log files. If version
+information is not included in the log files, you can instead add them in config.
+Software versions are specified in the YAML as pairs of software names and version
+strings. Multiple version of the same software are specified as lists. Example:
+
+```yaml
+software_versions:
+  samblaster: "0.1.24"
+  samtools:
+    - "1.15"
+    - "1.10"
+  some_other_tool: "2023-1"
```

Comment:
So on testing this samtools `1.10` was reported as `1.1` while `1.15` was actually fine. While it might be ok to leave out quotes in some cases from user standpoint I think it better to recommend passing strings here. At least until we have [proper config validation](https://github.com/ewels/MultiQC/issues/1892) inplace.

Reply:
Ah right, ok fair enough 👍🏻 

In that case, can we add a [warning admonition box](https://multiqc.info/docs/development/contributing/#warnings) pointing out this gotcha and saying to use quotes to be safe?